packages/react/src/components/header/components/navigationLink/navigationLinkDropdown/NavigationLinkDropdown.tsx

```diff
@@ -84,12 +89,12 @@ export const NavigationLinkDropdown = ({
   };
 
   const childElements = getChildElementsEvenIfContainersInbetween(children);
-
+  console.log(dropdownButtonClassName);
```

Comment:
I'm guessing this was left here unintentionally? 🤔

Reply:
Yes.. 🤦